docs/guide/ops/locations/location-customizers.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Location customizers
+layout: website-normal
+---
+
+Apache Brooklyn supports a number of ways to configure and customize locations. These include
+the `JcloudsLocationCustomizer`, which is for advanced customization of VM provisioning through jclouds.
+There is also a `MachineLocationCustomizer`, which allows customization of machines being obtained 
+from any kind of location (including Bring Your Own Nodes).
```

Comment:
Could link "Bring Your Own Nodes" to https://brooklyn.incubator.apache.org/v/latest/ops/locations/#byon.